.gitignore

```diff
@@ -48,3 +48,7 @@ nbdist/
 ### Komga
 /komga/src/main/resources/public/
 /komga/artemis/
+
+### Environment
+komga-webui/.env
+komga-webui/.env.development
```

Comment:
Can remove this if you like.  It's a habit of mine from other projects to include the .env in the ignore once it's initially committed to prevent other devs from accidentally committing theirs.

Reply:
note that like React, [you can use](https://cli.vuejs.org/guide/mode-and-env.html#environment-variables) `.local` files that you don't have to commit ;)

Reply:
Ahh it's more the habit of for work related projects the .env is always explicity ignored, so that other devs dont accidentally commit their environment settings and have it go to all other devs

Reply:
That why `.local` exists 😂